internal/page/fixtures/supporter.go

```diff
@@ -312,6 +312,7 @@ func waitForLPAIndex(searchClient *search.Client, organisationCtx context.Contex
 			PageSize: 1,
 		})
 		if err != nil {
+			log.Println("error waiting for LPA Index")
```

Comment:
So its immediately obvious what the stream of console errors are when this happens

Reply:
or could have the one below be `log.Println("error waiting for LPA index:", err)`